common/Reducers.jsm

```diff
@@ -484,6 +484,33 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
           spocs_endpoint: action.data || INITIAL_STATE.DiscoveryStream.spocs.spocs_endpoint,
         },
       };
+    case at.PLACES_LINK_BLOCKED:
+      // Return if action data is empty, or spocs or feeds data is not loaded
+      if (!action.data || !prevState.spocs.loaded || !prevState.feeds.loaded) {
+        return prevState;
+      }
+
+      return {
+        ...prevState,
+        spocs: {
+          ...prevState.spocs,
+          data: prevState.spocs.data.spocs ? {
+            spocs: prevState.spocs.data.spocs.filter(s => s.url !== action.data.url),
+          } : {},
```

Comment:
It seems like weird behaviour that that we'd go from `spocs.data.spocs` being falsey to `spocs.data.spocs` being `{}`

Reply:
Goal here is to reset spocs.data back to initial state of {} ( check prevState.spocs.data.spocs is at spocs.data level, I should indent to make it readable)